CONTRIBUTORS.md

```diff
@@ -6,96 +6,96 @@ This list is manually curated to include valuable contributions by volunteers th
 
 | GitHub Username | WordPress.org Username|
 | --------------- | --------------------- |
-| @youknowriad | |
-| @aduth | |
-| @jasmussen | |
-| @iseulde | |
-| @mtias | |
-| @nylen | |
-| @EphoxJames | |
-| @mkaz | |
-| @notnownikki | |
-| @BE-Webdesign | |
-| @njpanderson | |
-| @mimo84 | |
-| @intronic | |
-| @westonruter | |
-| @mcsf | |
-| @dmsnell | |
-| @afercia | |
-| @paulwilde | |
-| @mitogh | |
-| @codebykat | |
+| @youknowriad | @youknowriad |
+| @aduth | @aduth |
+| @jasmussen | @joen |
+| @iseulde | @iseulde |
+| @mtias | @mtias |
+| @nylen | @jnylen0 |
+| @EphoxJames | :mag: :tired_face: |
+| @mkaz | @mkaz |
+| @notnownikki | @notnownikki |
+| @BE-Webdesign | @chopinbach |
+| @njpanderson | :mag: :tired_face: |
+| @mimo84 | :mag: :tired_face: |
+| @intronic | :mag: :tired_face: |
+| @westonruter | @westonruter |
+| @mcsf | @mcsf |
+| @dmsnell | @dmsnell |
+| @afercia | @afercia |
+| @paulwilde | @paulwilde |
+| @mitogh | @mitogh |
+| @codebykat | @codebykat |
 | @ahmadawais | @mrahmadawais |
-| @kopepasah | |
-| @circlecube | |
-| @adamsilverstein | |
-| @timmyc | |
-| @ephox-mogran | |
-| @nb | |
-| @JDGrimes | |
-| @Soean | |
-| @mapk | |
-| @sirjonathan | |
-| @j-falk | |
-| @ryelle | |
+| @kopepasah | @kopepasah |
+| @circlecube | @circlecube |
+| @adamsilverstein | @adamsilverstein |
+| @timmyc | @timmydcrawford |
+| @ephox-mogran | :mag: :tired_face: |
+| @nb | @nbachiyski |
+| @JDGrimes | @JDGrimes |
+| @Soean | @Soean |
+| @mapk | @mapk |
+| @sirjonathan | @sirjonathan |
+| @j-falk | @j-falk |
+| @ryelle | @ryelle |
 | @ntwb | @netweb |
-| @lamosty | |
-| @willybahuaud | |
-| @maurobringolf | |
+| @lamosty | @lamosty |
+| @willybahuaud | @willybahuaud |
+| @maurobringolf | @maurobringolf |
 | @aaronjorbin | @jorbin |
-| @spocke | |
-| @androb | |
-| @annaephox | |
-| @Afraithe | |
-| @georgeh | |
-| @m | |
-| @melchoyce | |
-| @pento | |
-| @karmatosed | |
-| @nitrajka | |
-| @sirreal | |
-| @inhil | |
-| @georgeolaru | |
-| @martinlugton | |
-| @joyously | |
-| @rileybrook | |
-| @azaozz | |
-| @folletto | |
-| @ianstewart | |
-| @johnpixle | |
-| @mrwweb | |
-| @diegoliv | |
-| @lukecav | |
-| @shaunandrews | |
-| @hugobaeta | |
-| @mizejewski | |
-| @buzztone | |
-| @mathetos | |
-| @GaryJones | |
-| @jasonagnew | |
-| @brickbones | |
-| @iamgabrielma | |
-| @swissspidy | |
-| @dixitadusara | |
-| @ameeker | |
-| @StaggerLeee | |
-| @jblz | |
+| @spocke | @spocke |
+| @androb | @androb |
+| @annaephox | :mag: :tired_face: |
```

Comment:
There are several of these. Can they be removed, or the correct usernames be added?

Reply:
I am not able to find those (:mag: :tired_face:) usernames on WordPress.org with the help of full name, avatar, web URLs and also Google.

Reply:
I don't know how it can be solved.

Reply:
Nice work! If you can't find a wp.org account leave it empty. Maybe some of them don't have one.

Reply:
ok